client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/issue/ResolutionStatus.java

```diff
@@ -39,4 +39,13 @@ public String getTitle() {
   public String getDescription() {
     return description;
   }
+
+  public static ResolutionStatus[] get103orOlderStatuses() {
+    return new ResolutionStatus[] { ResolutionStatus.WONT_FIX, ResolutionStatus.FALSE_POSITIVE};
+  }
+
+  /** With SQ >= 10.4: "Won't fix" becomes "Accept" */
+  public static ResolutionStatus[] get104orNewerStatuses() {
+    return new ResolutionStatus[] { ResolutionStatus.ACCEPT, ResolutionStatus.FALSE_POSITIVE};
+  }
```

Comment:
I would not put those in this class, because it's not something we want to expose to clients. I would move them to the service. And you could return a List directly instead of using an array then converted to a list